Cargo.toml

```diff
@@ -23,6 +23,7 @@ version = "0.7.0"
 [workspace.dependencies]
 anyhow = { version = "1.0", default-features = false }
 assert_cmd = { version = "2.0.14", default-features = false }
+bytes = { version = "1.10.1", default-features = false }
```

Comment:
Already a dependency of reqwest, required to use the `Bytes` type used by reqwest but not exported.